### ISSUE TITLE
Question on Private Apps.

### ISSUE THREAD

**Issue by jhunexjun** [nozzlegear/ShopifySharp#812]
We're using this good software to connect to our Shopify. I just want to ask if we are affected with a notification that says:

   Some of your private apps and/or admin webhook subscriptions may not work as expected because they are using deprecated API versions.

  It also says, private apps need to be updated before January 1, 2023.

**Comment by nozzlegear** [comment]
Hey @jhunexjun! Which version of ShopifySharp are you using? If it's old enough you might be using a version of the Shopify API that is about to be deprecated. You can check our version support table in the readme here: https://github.com/nozzlegear/ShopifySharp#api-support

If that's not the problem, you probably just need to update your webhook api version. For some reason this is a manual process that you need to go into the app setup page on Shopify to do. There will be a dropdown to select the api version for your webhooks, I'd recommend using whichever version matches the API version of ShopifySharp.

<img width="654" alt="image" src="https://user-images.githubusercontent.com/2417276/206026342-28bd1886-78f8-4788-9511-391e2603850a.png">

Let me know if you have any issues!

**Comment by jhunexjun** [comment]
This is what I've got.

<img width="222" alt="image" src="https://user-images.githubusercontent.com/6801369/206273109-9542db69-b8e9-4a4a-89c6-f3003b1ee985.png">

I'm using v5.7.0.0. Do you think I have to move to newest version?

**Comment by jhunexjun** [comment]
@nozzlegear I checked Shopify Admin but we do not use Webhook. The App is only using API URL like `https://myshop.myshopify.com/admin/api/2020-07`.
What I did. I uninstalled ShopifySharp v5.7.0 then install ShopifySharp v5.16.0. After that I changed the API URL to `https://myshop.myshopify.com/admin/api/2022-04`. I also did small changes to `MetaField` property from `ValueType` to `Type` because the former is not found anymore. After that everything's looks great and running. No more changes to Shopify Admin.
